source/_patterns/02-organismer/30-seksjoner/31-personbytte.mustache

```diff
@@ -1,3 +1,10 @@
+<div class="a-dropdown-footer px-2 pt-3 {{ personswitch-isLeft }}">
```

Comment:
now that it is moved up, do we still need the 'a-dropdown-footer' class?

Reply:
good call. I debated that. I can remove it. I will update the pull-request,